app/components/events/view/export/download-zip.js

```diff
@@ -3,4 +3,9 @@ import Ember from 'ember';
 const { Component } = Ember;
 
 export default Component.extend({
+  actions: {
+    startGeneration() {
```

Comment:
Why this unnecessary intermediate action ? Remove it and use the passed in action directly,